aboutcitus/what_is_citus.rst

```diff
@@ -1,6 +1,41 @@
 What is Citus?
 ==============
 
-Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.
+Citus horizontally scales PostgreSQL across multiple machines using sharding and replication. It works well when you have a large data set and when you want to get answers from that data in human real-time – typically in less than a second.
 
 Citus extends the underlying database rather than forking it, which gives developers and enterprises the power and familiarity of a traditional relational database. As an extension, Citus supports new PostgreSQL releases, allowing users to benefit from new features while maintaining compatibility with existing PostgreSQL tools.
+
+When to Use Citus
+-----------------
+
+Example use cases:
+
+* Analytic dashboards with subsecond response times
+* Exploratory queries on unfolding events
+* Aggregate reports on large archives
+* Analyzing sessions with funnels, segmentation, and cohorts
+* Extending a data warehouse with real-time capabilities
+
+For concrete examples check out our customer `use cases <https://www.citusdata.com/solutions/case-studies>`_. Typical Citus workloads are operational, with aggregate queries and no long-lived transactions.
+
+Considerations for Use
```

Comment:
At a high level, I think this section makes sense. However, I think we can make it shorter and less repetitive.

eg. We say "Although Citus extends PostgreSQL with distributed functionality, it is not a drop-in replacement that scales out all workloads." and "PostgreSQL provides thousands of features, and Citus doesn’t yet scale them all."